oci/layer/tar_generate.go

```diff
@@ -201,7 +202,10 @@ func (tg *tarGenerator) AddFile(name, path string) error {
 	// XXX: This should probably be moved to a function in tar_unix.go.
 	names, err := tg.fsEval.Llistxattr(path)
 	if err != nil {
-		return errors.Wrap(err, "get xattr list")
+		if err != unix.EOPNOTSUPP {
```

Comment:
i think you want `errors.Cause()` here? something like this: https://github.com/opencontainers/umoci/blob/f74280bc7da2b5a7d0febd73b37278fe03fe6a17/oci/layer/tar_extract.go#L152-L154

Reply:
oh that's neat, thanks.